website/index.html

```diff
@@ -26,11 +26,6 @@ <h1>Finite Automata and Formal Languages</h1>
                 On this page, you can validate your input against the text formats used in the course
                 and test your answers to some of the questions.
             </p>
-            <p>
-                Note that passing these tests doesn't guarantee that your answer is correct;
-                you should still read the assignment carefully.
-                For example, this tool does not check for minimality.
-            </p>
```

Comment:
Some exercises ask for explanations in addition to the correct automaton. Maybe this is something we should comment on.